apolloschurchapp/src/index.js

```diff
@@ -109,11 +108,6 @@ const App = () => (
                 stackPresentation: 'push',
               })}
             />
-            <Screen
-              name="Event"
```

Comment:
should we put in a redirect from `Event` to content? I know if you try to navigate to a route that doesn't exist, you'll get an app crash